pkg/apis/serving/v1beta1/inference_service_status.go

```diff
@@ -90,8 +90,8 @@ const (
 	ExplainerRoutesReady apis.ConditionType = "ExplainerRoutesReady"
 	// PredictorConfigurationReady is set when predictor pods are ready.
 	PredictorConfigurationReady apis.ConditionType = "PredictorConfigurationReady"
-	// TransformerConfigurationeReady is set when transformer pods are ready.
-	TransformerConfigurationeReady apis.ConditionType = "TransformerConfigurationeReady"
+	// TransformerConfigurationReady is set when transformer pods are ready.
+	TransformerConfigurationReady apis.ConditionType = "TransformerConfigurationReady"
```

Comment:
API Break -- Can't be done. It's here because I'm fairly certain this is a typo. But to deal with it would require deprecating the old one instead of simply changing a published API.

Reply:
We should fix this and I think it should be fine as the conditions are array type updated with conditionType as the key, it should  be get inserted as a new condition element in the array but the wrong name might be still lingering around for existing services after the upgrade, might need some code for remediation to delete the old name when new name presents.

Reply:
Sounds complicated. If someone can talk me through this on Slack, I'm happy to try.

Otherwise I'm tempted to drop it and file an issue for someone else to work through...

Reply:
@jsoref Please help drop this change and create an issue, we will take care of the change for this. Thanks!